randovania/games/dread/json_data/Artaria.txt

```diff
@@ -1113,7 +1136,9 @@ Extra - asset_id: collision_camera_010
   > Door to Wide Beam Block Room (DoorFrame_006)
       Trivial
   > Door to EMMI Zone Ballspark Hallway
-      Spider Magnet or Space Jump or Simple IBJ
+      Any of the following:
+          Spider Magnet or Space Jump or Speed Booster or Simple IBJ
+          Flash Shift and After Artaria - Central Unit and Walljump (Advanced)
```

Comment:
I personally put any single wall jumps with flash as intermediate no matter how tall (except if only one wall jump is needed)